<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wind</class>
 <widget class="QWidget" name="wind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>213</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>状态栏</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>高度：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>400</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>初速度：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>速度损失系数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>0.6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>小球半径：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>速度级别</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开始演示</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>继续</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
